package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -43,12 +48,15 @@ local s = f_password:section(Section)
 
 local pw1 = s:option(Value, "pw1", translate("Password"))
 pw1.password = true
+pw1.datatype = 'minlength(12)'
```

Comment:
After thinking about this a bit more, I'm okay with making the limit configurable in site.conf after all.